include/madrona/viz/viewer.hpp

```diff
@@ -34,6 +34,12 @@ class Viewer {
         ExecMode execMode;
     };
 
+    struct VoxelConfig {
```

Comment:
Let's just put these params at the end of the normal Viewer::Config with default values 0 so people not using the voxels don't need to pass something here.